conference/react-js/src/containers/SignUpForm/SignUpPopUp/RegistrationInputBox.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const RegistrationInputBox = props => (
```

Comment:
I am sure we have some component named `InputBox`. Can we extend it instead of creating new one? Or maybe there is a possibility to reuse it?

Reply:
done